code/src/components/Dropup.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import data from '../stretch-goal.json'
+
+const { useState } = React;
```

Comment:
impressive you guys did all the stretch goals so cool!

Reply:
I agree with Jennie on this, really nice work!